perceval/backends/core/jira.py

```diff
@@ -96,8 +96,8 @@ class Jira(Backend):
 
     :param url: JIRA's endpoint
     :param project: filter issues by project
-    :param user: Jira user
-    :param password: Jira user password
```

Comment:
I would revert this change because the current implementation is based on password. It is better if we change it only after the implementation is changed to token based.

Reply:
When I used Jira backend in my project, I use email for the user field and api token for the password field. However, it did not work when I used email and password for authentication. Should I still revert this change?

Reply:
Since the current implementation is based on password, changing the comment to token might not look good. It would be really great if you can work on the feature of changing the basic_auth to token_auth. You can work in the same PR. WDYT?